cli/src/main.rs

```diff
@@ -67,6 +67,7 @@ pub async fn main() {
     .subcommand(SubCommand::with_name("new")
       .about("Deploys your repository to a new app with one of the deploy configs at ~/.anycloud/deploy.json")
       .arg_from_usage("<DEPLOY_NAME> 'Specifies the name of the deploy config to use'")
+      .arg_from_usage("<APP_ID> 'Specifies the application identifier'")
```

Comment:
this is optional and if one is not defined then we generate it for the user

Reply:
[Here's an example of how to make an optional arg in clap](https://github.com/alantech/alan/pull/439/files#diff-4a93dc22cf1d945b9f18ed83a68209c3baf7601ac232d6bb320e133996aba81bR80)